Automattic/jetpack
# Contact Form: Update Form block placeholder to include the patterns modal

Fixes #27328

#### Changes proposed in this Pull Request:
* Update Form block placeholder to include the patterns modal
* The Patterns modal should show only patterns that contain the `jetpack/contact-form` block

![Screen Shot 2022-11-09 at 11 53 02](https://user-images.githubusercontent.com/7811225/200927920-47fbcb95-441b-4f32-bab8-0ba7ffa41413.png)
 
#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Create a Post/Page and add a Form block
* Check if the Form block placeholder has the "Explore Forms Patterns" button
* Click the button and check if the modal shows only form related patterns
